po/Makefile

```diff
@@ -1,7 +1,7 @@
 all:
 .PHONY: update-po update-js update-pot
 
-LINGUAS := cs de el es ko nl pl sv tr zh_CN zh_TW
+LINGUAS := ru_RU cs de el es ko nl pl sv tr zh_CN zh_TW
```

Comment:
this should simply be "ru" I think

Reply:
When just "ru" that does not work translation in the Russian locale.

Reply:
I checked out your code and tested here, and I can't reproduce your problem. For me it's the other way around..

Our code is written so that if the browser asks for a specific sub-language like 'ru_RU' and no such specific translation exists, it will fallback to use 'ru'. If we name the translation like 'ru_RU' and the browser asks for 'ru' we will not get a translation.

Reply:
Could you please also place it alphabetically in this list?